### PR TITLE
Allowing dot notation in get

### PR DESCRIPTION
**Pull request by assertchris**

### CODE REVIEW

File: src/AbstractContainer.php [hunk 1]
```diff
@@ -177,6 +177,11 @@ public function get()
         $path = func_get_args();
         $key  = implode('.', $path);
 
+        // allow dot-notation in first parameter
+        if (count($path) === 1 && strstr($path[0], '.')) {
+            $path = explode('.', $path[0]);
+        }
```
Comment (rtheunissen) [inline src/AbstractContainer.php:183]: See https://github.com/rtheunissen/config/issues/1.

~~`implode('.', $path)` takes care of it already, doesn't it?~~

Could we use `strpos($path[0], '.') !== false` instead?

Another minor thing is to capitalise the first letter on an inline comment to be consistent with the rest of the project. `Allow dot-notation...`

File: tests/AbstractContainerTest.php [hunk 2]
```diff
@@ -246,5 +246,32 @@ public function testInvalidGet($types, $defaults, $instances, $key=null)
...
+        $this->assertEquals(123, $container->get('foo.bar'));
+
+        $container = MockContainer::make([
+            'foo' => [
+                'bar' => 456,
+            ],
+        ]);
+
+        $this->assertEquals(456, $container->get('foo.bar'));
+    }
```
Comment (rtheunissen) [inline tests/AbstractContainerTest.php:275]: :+1:

File: src/AbstractContainer.php (hunk 1 above)
Comment (assertchris) [inline src/AbstractContainer.php:183]: done!

**Comment by rtheunissen** [comment]
:space_invader:

**Pull request merged**